packages/scraper-tools/src/browser_tools.ts

```diff
@@ -100,6 +108,7 @@ export interface DumpConsoleOptions {
  * Chromium messages, usually internal errors, occuring in page.
  */
 export function dumpConsole(page: Page, options: DumpConsoleOptions = {}) {
+    // @ts-expect-error Parameter 'msg' implicitly has an 'any' type. // TODO
     page.on('console', async (msg) => {
```

Comment:
just add the type (ideally put there something more strict than `any`)
```suggestion
    page.on('console', async (msg: any) => {
```

Reply:
it would work for example with JSHandle, but it would bring the puppeteer dependency, while we don't really want that here :/

Reply:
i dont care much about the type, this is more about "lets not overuse ts-ignore comments". i am fine with `msg: any`, more than with the ignore comment :]